frontend/components/doctor/patient-info-card.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Image, Divider, Text, Flex } from "@chakra-ui/react";
+import { Patient } from "@frontend/models/patient";
+import PatientStatus from "./patient-status";
+
+interface AppProps {
+    patient: Patient;
+}
+
+export default function PatientInfoCard({ patient }: AppProps) {
```

Comment:
since AppProps only has one prop, it could be replaced by `PatientInfoCard({ patient }: {patient: Patient)` instead